libs/designer-ui/src/lib/displayText/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from '@fluentui/react';
+
+export interface DisplayTextEditorProps {
+  displayTextText: string;
+  relativeLinkText?: string;
+  relativeLink?: string;
+  openRelativeLink?: (relativeLink: string) => void;
+}
+
+export const DisplayTextEditor: React.FC<DisplayTextEditorProps> = ({
+  displayTextText,
+  relativeLinkText,
+  relativeLink,
+  openRelativeLink,
+}): JSX.Element => {
+  return (
+    <div className="msla-displayText-editor-container">
+      {displayTextText ?? ''}
+      <Link className="msla-workflow-parameters-link-text" onClick={() => openRelativeLink?.(relativeLink ?? '')}>
```

Comment:
NIT: I think that despite the fact that this `msla-workflow-parameters-link-text` class works for this component, maybe a good idea would be to not use it because the DisplayTextEditor can be use not only in parameters tab